lib/floe/boolean.rb

```diff
@@ -0,0 +1,10 @@
+module Floe
+  module Boolean
```

Comment:
Not sure I like this here - Originally you had it under Floe::Workflow::IntrinsicFunction::Boolean, which made more sense to keep it contained there.  Also, not sure I like messing with the internals of TrueClass and FalseClass.

Would like another opinion though - @agrare ?